Modules/Scripted/QReads/Resources/UI/QReads.ui

```diff
@@ -504,6 +504,31 @@ QPushButton#ResetWLButton {
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QPushButton" name="HelpButton">
+     <property name="font">
+      <font>
+       <family>MS Shell Dlg 2</family>
```

Comment:
Are you sure to change the font here? Instead I suggest to apply this type of changes through the stylesheet